### PR TITLE
Renamed writeEventsToStream method into appendEventsToStream

### PR DESCRIPTION
**Pull request by oskardudycz**
Renamed `writeEventsToStream` method into `appendEventsToStream` to be aligned with the .NET Client convention and general Event Sourcing convention (as we're appending events at the end of the stream).

I'd also suggest removing `event` from the name as .NET Client does not use it. I like the current convention in this repo, but .NET client is already used, and we cannot break it. I think that it's better to keep consistency. I didn't include that change, as I wanted to have the consensus before doing that.  If we agreed on that - I can send a follow-up PR.

### CODE REVIEW

File: src/streams/index.ts [hunk 1]
```diff
@@ -4,4 +4,4 @@ export * from "./readEventsFromStream";
 export * from "./subscribeToAll";
 export * from "./subscribeToStream";
 export * from "./tombstoneStream";
-export * from "./writeEventsToStream";
+export * from "./appendEventsToStream";
```
Comment (George-Payne) [inline src/streams/index.ts:7]: Index exports should be alphabetical, but its not enforced and probably not followed everywhere

**Review by George-Payne — APPROVED** [review]
Very thorough

File: src/streams/index.ts (hunk 1 above)
Comment (oskardudycz) [reply src/streams/index.ts]: Updated in: https://github.com/EventStore/EventStore-Client-NodeJS/pull/88/commits/b3ea9b4236373d9c58dcd2839977ab9041482eb9.

**Comment by alexeyzimarev** [comment]
If we want to rename functions for the purpose of aligning the naming, we should go for it and me more consistent. For example, we should name this function `appendToStream` and remove the name `Event` from there as we cannot possibly know what people append to streams (events, commands, snapshots, whatever else).

**Comment by oskardudycz** [comment]
Closing this one as I'll merge the changes in #90

**Pull request closed without merging**